test/protractor.conf.js

```diff
@@ -89,7 +89,9 @@ exports.config = {
   // run.
   suites: {
     // smoke: 'spec/smoketests/*.js',
-    rest_api: 'integration/rest_api/*.js' // This suite of tests should only require a running master api, it should not require a node
+    'add-to-project': 'integration/features/user_adds_template_to_project.spec.js',
+    'create-project': 'integration/features/user_creates_project.spec.js' // This suite of tests should only require a running master api, it should not require a node
```

Comment:
ok its fine for now but if we start getting a bunch of these we may want to organize them